### DISCUSSION TITLE
[Feature] iOS Burst Photo support

### DISCUSSION THREAD

**Discussion by madjam002** [immich-app/immich#1770] [Feature Request]
### The feature

Currently iOS Burst Photos just upload a single image from the burst rather than all of the images that are part of the burst.

When using an app like PhotoSync, all of the images are uploaded individually. Perhaps with Immich an initial implementation would be to just show the images individually, then in the future they could be grouped together?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
Do you know if BurstPhoto is only available on iPhone 14? So you have a set of burst photo you can share for testing?

**Reply by madjam002** [reply]
It's been around for a while, I had it at least from my iPhone 6, maybe even earlier...

(photos shared on Discord)

It looks like there is some EXIF metadata that joins the burst photos together
```
Burst UUID                      : 2C3C071A-BC9C-4EE0-86DF-B72E341B784A
```

**Reply by t3chn0m4g3** [reply]
Using Immich, but this is actually the only reason why I still have to backup photos with PhotoSync. Supporting burst mode would be a great feature!

**Comment by samssausages** [comment]
+1 to this, having the same problem and missing out on a lot of pictures being backed up due to this.

Most people probably don't realize this isn't backing up unless they specifically look for it.

Pretty sure burst is an option on Android as well, but the implementation is likely different.

**Comment by EliasMarine** [comment]
I would love for the burst photo stack to become a feature on mobile and web app. This is how the other photo app does it...

![burst_1](https://github.com/immich-app/immich/assets/95382778/ed35e159-d38f-4efd-b350-1c784bffa7ca)
![burst_2](https://github.com/immich-app/immich/assets/95382778/853bc9b5-ce24-4f4e-a278-c18a5d8247de)

**Comment by melyux** [comment]
Can't believe this isn't supported yet. As far as "backing up" goes, Immich does a pretty poor job even in 2026 (not uploading bursts, not uploading captions, not uploading originals of edited pictures), things PhotoSync has done perfectly for years.

**Comment by BradNagel** [comment]
I just noticed this issue when I did a mass device delete of burst shots I did for a Baseball game. It notified me that they are burst photos and what will happen when I delete them but was confused why it had not backed them up if it knew they were burst photos.
